src/txmempool.h

```diff
@@ -334,8 +334,11 @@ struct TxMempoolInfo
     /** Time the transaction entered the mempool. */
     int64_t nTime;
 
-    /** Feerate of the transaction. */
-    CFeeRate feeRate;
+    /** Fee of the transaction. */
+    CFeeRate fee;
+
+    /** Virtual size of the transaction. */
+    size_t vsize;
```

Comment:
How this variable is supposed to be assigned?

Reply:
here https://github.com/bitcoin/bitcoin/pull/16507/files#diff-ca74c4b28865382863b8fe7633a85cd6R776

Reply:
Yikes, that looks dangerous. What would happen if code without this PR got mixed in with the PR'd type?

Reply:
whoops! `fee` is supposed to be `CAmount`.